.github/labeler.yml

```diff
@@ -3,3 +3,16 @@
 dependencies:
   - '**/*requirements*.*'
   - '**/setup.py'
+demo:
+  - any: ['demos/**/*', '!demos/**/*.md']
+documentation:
+  - '*.md'
+  - '**/*.md'
+model:
+  - 'models/**/*.yml'
```

Comment:
not sure that it is correct.
now, label model will be added only if you modify model.yml or accuracy-check.yml, but the second may have more relation to AC, and what about model documentation and extra files vocabs?

Reply:
e.g. for intel trained models we can not have model.yml at all when it will be added manually before package uploaded